src/jdk.compiler/share/classes/com/sun/source/util/DocTreeScanner.java

```diff
@@ -500,6 +500,21 @@ public R visitSince(SinceTree node, P p) {
      * @param node  {@inheritDoc}
      * @param p  {@inheritDoc}
      * @return the result of scanning
+     * @since 18
+     */
+    @Override
+    public R visitSnippet(SnippetTree node, P p) {
+        R r = scan(node.getAttributes(), p);
+        r = scanAndReduce(node.getBody(), p, r);
+        return r;
+    }
+
+    /**
+     * {@inheritDoc} This implementation scans the children in left to right order.
+     *
+     * @param node  {@inheritDoc}
+     * @param p  {@inheritDoc}
```

Comment:
I don't think that I need `{@inheritDoc}` to inherit the comments in that and the similar cases in that file. I used that style because other parts of the file use it and I didn't want to update the file inconsistently. If we don't like that style, we are free to change it. That said, let's NOT do it in this PR.